src/Security/ConnectPassport.php

```diff
@@ -12,14 +12,15 @@
 namespace SymfonyCorp\Connect\Security;
 
 use Symfony\Component\Security\Core\User\UserInterface;
+use Symfony\Component\Security\Http\Authenticator\Passport\Passport;
 use Symfony\Component\Security\Http\Authenticator\Passport\PassportTrait;
 use Symfony\Component\Security\Http\Authenticator\Passport\UserPassportInterface;
 use SymfonyCorp\Connect\Api\Entity\User;
 
 /**
  * @author Fabien Potencier <fabien@symfony.com>
  */
-class ConnectPassport implements UserPassportInterface
+class ConnectPassport extends Passport implements UserPassportInterface
```

Comment:
It should stop implementing the deprecated interface directly also, don't forget the `use` statement :)

```suggestion
class ConnectPassport extends Passport
```

Reply:
Thanks for the review! I've removed the interface.